### PR TITLE
Bumps brotobufjs from 6.8.8 to 6.11.3

### PR DESCRIPTION
**Pull request by xuesichao**
**Issue #:**
Current version `protobufjs` has high severity issue, we need to upgrade it to the latest version to be able to successfully run `npm run build:release`.

**Description of changes:**
* Bump `protobufjs` from 6.88 to 6.11.3.
* Implement the missing property `stream` required by type `Blob` in `DOMBolbMock`.
* Add back the `npm audit` step in `prebuild` script

**Testing:**

*Can these tested using a demo application? Please provide reproducible step-by-step instructions.*
N/A

**Checklist:**

1. Have you successfully run `npm run build:release` locally?
Yes.

2. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
N/A

3. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
Update the `protobufjs` dependency that is used in our wire protocol. However the wire compatibility will never be broken by version change so that is fine.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/classes/getusermediaerror.html [hunk 1]
```diff
@@ -112,7 +112,7 @@ <h3>Properties</h3>
 								<li class="tsd-kind-property tsd-parent-kind-class"><a href="getusermediaerror.html#cause" class="tsd-kind-icon">cause</a></li>
 								<li class="tsd-kind-property tsd-parent-kind-class tsd-is-inherited"><a href="getusermediaerror.html#message" class="tsd-kind-icon">message</a></li>
 								<li class="tsd-kind-property tsd-parent-kind-class tsd-is-inherited"><a href="getusermediaerror.html#name" class="tsd-kind-icon">name</a></li>
-								<li class="tsd-kind-property tsd-parent-kind-class tsd-is-overwrite tsd-is-inherited"><a href="getusermediaerror.html#stack" class="tsd-kind-icon">stack</a></li>
+								<li class="tsd-kind-property tsd-parent-kind-class tsd-is-inherited"><a href="getusermediaerror.html#stack" class="tsd-kind-icon">stack</a></li>
```
Comment (ltrung) [inline docs/classes/getusermediaerror.html:115]: Why is there doc change?

Comment (xuesichao) [reply docs/classes/getusermediaerror.html:115]: No idea, it appears after running `npm run build:release` to generate the doc.

Comment (ltrung) [reply docs/classes/getusermediaerror.html:115]: Can you track down what change causes it?

File: test/domblobmock/DOMBlobMock.ts [hunk 2]
```diff
@@ -20,8 +20,11 @@ export default class DOMBlobMock implements Blob {
     throw new Error('Unimplemented.');
   }
 
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  stream(): ReadableStream<any> {
+  stream(): NodeJS.ReadableStream;
```
[the remark this answers is not in the document]
Comment (xuesichao) [reply test/domblobmock/DOMBlobMock.ts:23]: Yes. `protobufjs` requires a higher version of `@types/node`. There're changes to type `Blob`. So need to implement the missing `stream` property of different type. I mentioned in the description.

File: docs/classes/getusermediaerror.html (hunk 1 above)
Comment (xuesichao) [reply docs/classes/getusermediaerror.html:115]: It's due to the type change of Error. Because we update the `@types/node`.

Comment (xuesichao) [reply docs/classes/getusermediaerror.html:115]: Btw do you think upgrading this `@types/node` is a breaking change?

File: package-lock.json [hunk 3]
```diff
@@ -5526,9 +5526,9 @@
       "dev": true
     },
     "@types/node": {
-      "version": "10.17.5",
-      "resolved": "https://registry.npmjs.org/@types/node/-/node-10.17.5.tgz",
-      "integrity": "sha512-RElZIr/7JreF1eY6oD5RF3kpmdcreuQPjg5ri4oQ5g9sq7YWU8HkfB3eH8GwAwxf5OaCh0VPi7r4N/yoTGelrA=="
+      "version": "17.0.38",
```
Comment (xuesichao) [inline package-lock.json:5529]: @simmkyu @ltrung , will this `upgrading @types/node from 10.17.5 to 17.0.38` be considered as a breaking change?

Comment (xuesichao) [reply package-lock.json:5529]: `protobufjs@6.11.3` requires `@types/node": ">=13.7.0`, previously `protobufjs@6.8.8` requires `@types/node": "^10.1.0`

**Review by ltrung — APPROVED** [review]

**Pull request merged**